server/app/controllers/TransactionsController.scala

```diff
@@ -23,10 +20,11 @@ class TransactionsController @Inject() (
   def getTransactions(
       limit: Int,
       lastSeenTxid: Option[String],
-      orderingCondition: String
+      orderingCondition: String,
+      includeZeroTransactions: Boolean
   ) = public { _ =>
     transactionService
-      .get(Limit(limit), lastSeenTxid, orderingCondition)
+      .get(Limit(limit), lastSeenTxid, orderingCondition, includeZeroTransactions)
```

Comment:
Add a test for the new argument

Reply:
not sure i can add a meaningful test for that since the controller tests have the TransactionDataHandler mocked, or what did you have in mind?

Reply:
Fair enough